depend/secp256k1/Makefile.am

```diff
@@ -172,10 +178,6 @@ if ENABLE_MODULE_ECDH
 include src/modules/ecdh/Makefile.am.include
 endif
 
-if ENABLE_MODULE_SCHNORR
```

Comment:
seems like schnorr support was removed, do we use it though?

Reply:
afaik we do not, but maybe @svyatonik can help with that too?

Reply:
Ehhh...The only thing I can guarantee is that we don't use it in blockchain clients - parity-ethereum (and specifically in SecretStore), parity-bitcoin, parity-zcash (and obviously in substrate). Can't tell about the the rest...